spring-cloud-starter-stream-processor-groovy-transform/src/test/resources/script-with-grab.groovy

```diff
@@ -0,0 +1,3 @@
+@Grab('org.grooscript:grooscript:1.3.0')
+import org.grooscript.GrooScript
```

Comment:
to be honest you even don't need `import`: just `org.grooscript.GrooScript.convert(new String(payload))` is enough!

Reply:
test won't pass without import.

Reply:
oh, I see, you qualified the GrooScript, sure will do the same if that is preferred.

Reply:
causing error still....